docs/user_guide/SIMP_Administration/Environments/Deployment_Scenarios.inc

```diff
@@ -1,64 +1,143 @@
+.. |remote_mods__link|  replace:: Remote (Git hosting service)
+.. |r10k_de| replace:: ``r10k deploy environment``
+.. |r10k_pi| replace:: ``r10k puppetfile install``
+.. |r10k_pcd| replace:: ``puppet code deploy``
+
+..  ### Images
+
+.. |r10k_local_repos__no_control_repo| image:: ../../images/diagrams/deployment_scenarios/r10k_local_repos__no_control_repo.png
+    :alt: r10k + local, SIMP RPM-delivered module repositories
+.. |r10k__control_repo__manual| image:: ../../images/diagrams/deployment_scenarios/r10k__control_repo__manual.png
+    :alt: r10k + Control Repository (manual deploy)
+.. |r10k__control_repo__webhook| image:: ../../images/diagrams/deployment_scenarios/r10k__control_repo__webhook.png
+    :alt: r10k + Control Repository (webhook deploy)
 .. |enterprise__automated_deploy| image:: ../../images/diagrams/deployment_scenarios/enterprise__automated_deploy.png
     :alt: PE Code Manager + Control Repository
 
-Local: r10k + SIMP RPM-delivered module repositories
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Deployment Scenarios
+--------------------
 
-.. NOTE::
+SIMP supports deploying code using r10k or Code Manager, even when its Puppet
+modules are installed from RPMs.  However, site practices vary widely—the
+optimal practices for deploying SIMP will vary, depending on your deployment
+arcitecture.
+
+.. table:: Deployment Scenarios
+   :widths: 7 10 10 3 7 3
+
+   +------------+----------------------+------------+-------------+---------------------+---------+
+   | Deployment | Variation            | r10k       | Control     | Module Repos        | Auto?   |
+   | Scenario   |                      | command    | Repository? |                     |         |
+   +============+======================+============+=============+=====================+=========+
+   | Local      | |local_dsc__link|    | |r10k_pi|  | No          | |local_mods__link|  |         |
+   +------------+----------------------+------------+-------------+---------------------+---------+
+   | Control    | |crepo_dsc__link|    | |r10k_de|  | Yes         | |remote_mods__link| |         |
+   | Repository +----------------------+------------+             |                     +---------+
+   |            | |wh_crepo_dsc__link| | |r10k_de|  |             |                     | |check| |
+   |            +----------------------+------------+             |                     +---------+
+   |            | |pe_crepo_dsc__link| | |r10k_pcd| |             |                     | |check| |
+   +------------+----------------------+------------+-------------+---------------------+---------+
+
+.. _ug-sa-env-deployment-scenarios--local:
+
+Local deployment scenarios
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Local deployment scenarios will work out-of-the-box on systems where SIMP
+has been installed by ISO or RPM.  You don't need a control repository, a
+remote Git-hosting service, or even a functioning network
+
+relies on SIMP RPM-created Puppet module Git repository
```

Comment:
This looks like it is missing something.  Should it just be "It releis on SIMP RPM..."  or was there something else you were going to say.

Reply:
:ballot_box_with_check: Good catch; I've re-written that section